helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -111,8 +111,6 @@ processors:
   {{- include "splunk-otel-collector.otelMemoryLimiterConfig" .Values.otelAgent | nindent 2 }}
 
   batch:
```

Comment:
I remember facing issues with null values in helm. Can we set `batch: {}` instead?

Reply:
AFAIR you cannot override them in user's values files. Feel free to try, I may be wrong

Reply:
Seems to work fine merging in values from otelAgent.processors.batch. We do the same thing with zpages extension that was already there so I think you're thinking of a different issue.

Reply:
I think you might be thinking of a situation where in values.yaml we had to do `foo: {}` instead of `foo:` due to some reason.

Reply:
Ok, maybe. I don't remember exact reason